frontend/src/pages/SubProjects/SubprojectDialog.js

```diff
@@ -18,9 +18,7 @@ const handleCreate = props => {
     projectedBudgets.map(b => ({ ...b, value: fromAmountString(b.value).toString(10) }))
   );
   onDialogCancel();
-  storeSnackbarMessage(
-    strings.common.added + " " + strings.common.subproject + " " + shortenedDisplayName(displayName)
-  );
+  storeSnackbarMessage(strings.subproject.subproject_permissions_warning + " " + shortenedDisplayName(displayName));
```

Comment:
does this mean the success message is not displayed at all?

Reply:
that's right, since it would be too much info at once for the user, I thought one message is enough